README.md

```diff
@@ -13,9 +13,12 @@ A reference implementation and backport of background workers and tasks in Djang
 ## Installation
 
 ```
-pip install django-tasks
+python -m pip install django-tasks
 ```
 
+> [!TIP]
+> Add an extra name for each database you want to use (e.g. `django-tasks[mysql]`,  `django-tasks[postgres]` or `django-tasks[mysql,postgres]`).
+
```

Comment:
Nit: I don't think we should document this here. These extras are really more for dev. A user should have installed these for themselves. There's no runtime dependency on these libraries.

Reply:
I understand I'll remote it from README